src/bindings/python/CMakeLists.txt

```diff
@@ -455,204 +457,37 @@ if(WITH_CHECK)
-            ${CMAKE_SOURCE_DIR}/examples/sample-models/from-spec/level-3/enzymekinetics.xml
-        )
-        ADJUST_PYTHONPATH(test_python_printUnits)
-    
-        add_test(NAME test_python_promoteParameters
-            COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/examples/python/promoteParameters.py
-            ${CMAKE_SOURCE_DIR}/examples/sample-models/from-spec/level-3/enzymekinetics.xml
-            promoteParameters.out.xml
-        )
-        ADJUST_PYTHONPATH(test_python_promoteParameters)    
-    
-        add_test(NAME test_python_readSBML
-            COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/examples/python/readSBML.py
-            ${CMAKE_SOURCE_DIR}/examples/sample-models/from-spec/level-3/enzymekinetics.xml
-        )
-        ADJUST_PYTHONPATH(test_python_readSBML)    
-        
-        add_test(NAME test_python_replaceOneFD
-            COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/examples/python/replaceOneFD.py
-            ${CMAKE_SOURCE_DIR}/examples/sample-models/from-spec/level-3/functiondef.xml
-            f
-            reaction_1
-            replaceOneFD.out.xml
-        )
-        ADJUST_PYTHONPATH(test_python_replaceOneFD)    
-        
-        add_test(NAME test_python_setIdFromNames
-            COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/examples/python/setIdFromNames.py
-            ${CMAKE_SOURCE_DIR}/examples/sample-models/from-spec/level-3/enzymekinetics.xml
-            setIdFromNames.out.xml
-        )
-        ADJUST_PYTHONPATH(test_python_setIdFromNames)    
-        
-        add_test(NAME test_python_unsetAnnotation
-            COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/examples/python/unsetAnnotation.py
-            ${CMAKE_SOURCE_DIR}/examples/sample-models/from-spec/level-3/enzymekinetics.xml
-            unsetAnnotation.out.xml
-        )
-        ADJUST_PYTHONPATH(test_python_unsetAnnotation)    
-        
-        add_test(NAME test_python_unsetNotes
-            COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/examples/python/unsetNotes.py
-            ${CMAKE_SOURCE_DIR}/examples/sample-models/from-spec/level-3/enzymekinetics.xml
-            unsetNotes.out.xml
-        )
-        ADJUST_PYTHONPATH(test_python_unsetNotes)    
-        
-        add_test(NAME test_python_validateSBML
-            COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/examples/python/validateSBML.py
-            ${CMAKE_SOURCE_DIR}/examples/sample-models/from-spec/level-3/enzymekinetics.xml
-        )
-        ADJUST_PYTHONPATH(test_python_validateSBML)    
+        # Example of a test
+        # add_test(NAME test_python_addCVTerms
+        #     COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/examples/python/addCVTerms.py
+        #     ${CMAKE_SOURCE_DIR}/examples/sample-models/from-spec/level-3/enzymekinetics.xml
+        #     addCVTerms.out.xml
+        # )
+        # ADJUST_PYTHONPATH(test_python_addCVTerms)
+
+        message("NO PYTHON TESTS DEFINED YET")
```

Comment:
What python tests are we reporting then?

Reply:
We are testing the SWIG/Python interface library that we build.